src/core/Akka/Configuration/ConfigurationFactory.cs

```diff
@@ -126,6 +140,18 @@ public static Config FromResource(string resourceName, Assembly assembly)
                 }
             }
         }
+
+        /// <summary>
+        /// Creates a configuration based on the supplied source object
+        /// </summary>
+        /// <param name="source">The source object</param>
+        /// <returns>The configuration created from the source object</returns>
+        public static Config FromObject(object source)
+        {
+            var json = JsonConvert.SerializeObject(source);
+            var config = ParseString(json);
+            return config;
```

Comment:
This lets us create Hocon Configs from typed objects.

a convenience method only, but opens up for using objects as includes, or creating typed configuration support.